scripts/parse_skill_args.py

```
@@ -0,0 +1,72 @@
+    if match:
+        return match.group(1).strip()
+
+    return ""
+
+
+def main():
+    """CLI entry point."""
+    if len(sys.argv) != 3 or not sys.argv[1].startswith("--"):
+        exit_error("Usage: python scripts/parse_skill_args.py --flag-name <args_string>")
+
+    flag_name = sys.argv[1].lstrip("--")
```

Ruff (0.16.1)

[warning] Using `.strip()` with multi-character strings is misleading

(B005)